.github/workflows/build-test.yaml

```diff
@@ -2612,8 +2612,8 @@ jobs:
         run: |
           set +e
 
-          ./bin/kots admin-console push-images \
```

Comment:
let's not update this test. this is not the functionality that should be using the new `copy-public-images` command

Reply:
I didn't actually look at what the test was for, just assumed it was trying to push the public images since the provided airgap file didn't exist. Instead it's trying to go far enough that the registry check happens, I think I've fixed it, but we'll see when it runs